cmd/broker/main_suite_test.go

```diff
@@ -64,6 +64,6 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	servicePath = filepath.Join(absPath, "services")
 })
 
-var _ = AfterEach(func() {
+var _ = AfterSuite(func() {
 	os.RemoveAll(configPath)
 })
```

Comment:
per-suite means one config for all tests, right?

Reply:
for all tests running on a given node. 

this prevents having port collisions when parallel nodes try to run integration tests.

Reply:
what's a node?